test/feature/StrongMode/Ellision.js

```diff
@@ -0,0 +1,15 @@
+// Options: --strong-mode
+
+'use strong'
+
+let a = [0, 1, 2, ];
+assert.equal(3, a.length);
+
+let [, b, , c] = [0, 1, 2, 3];
+assert.equal(1, b);
+assert.equal(3, c);
+
+// Not working ATM.
+// let d;
+// [, d] = [0, 1];
```

Comment:
Hm, good point. We could also choose to make elisions illegal in patterns. Moving forward, I'd rather have proper wildcard patterns than relying on the quirky array syntax. But of course, we don't have that yet. Still, we might opt for the conservative (and simpler) choice. Programmers would have to invent dummy variables for now. Is it a common enough idiom to worry about?